domain/src/main/java/org/oppia/android/domain/profile/ProfileManagementController.kt

```diff
@@ -33,29 +33,31 @@ import java.util.Locale
 import javax.inject.Inject
 import javax.inject.Singleton
 
-private const val TRANSFORMED_GET_PROFILES_PROVIDER_ID = "transformed_get_profiles_provider_id"
-private const val TRANSFORMED_GET_PROFILE_PROVIDER_ID = "transformed_get_profile_provider_id"
-private const val TRANSFORMED_GET_WAS_PROFILE_EVER_ADDED_PROVIDER_ID =
-  "transformed_was_profile_ever_added_provider_id"
-private const val TRANSFORMED_GET_DEVICE_SETTINGS_PROVIDER_ID =
-  "transformed_device_settings_provider_id"
-private const val ADD_PROFILE_TRANSFORMED_PROVIDER_ID = "add_profile_transformed_id"
-private const val UPDATE_NAME_TRANSFORMED_PROVIDER_ID = "update_name_transformed_id"
+private const val GET_PROFILES_PROVIDER_ID = "get_profiles_provider_id"
+private const val GET_PROFILE_PROVIDER_ID = "xget_profile_provider_id"
+private const val GET_WAS_PROFILE_EVER_ADDED_PROVIDER_ID =
+  "was_profile_ever_added_provider_id"
+private const val GET_DEVICE_SETTINGS_PROVIDER_ID =
+  "device_settings_provider_id"
+private const val ADD_PROFILE_PROVIDER_ID = "add_profile_provided_id"
+private const val UPDATE_NAME_PROVIDER_ID = "update_name_provider_id"
 private const val UPDATE_PIN_TRANSFORMED_PROVIDER_ID = "update_pin_transformed_id"
```

Comment:
Update this to 
`private const val UPDATE_PIN_PROVIDER_ID = "update_pin_id"`

Reply:
I think it should be 

private const val UPDATE_PIN_PROVIDER_ID = "update_pin__povider_id"

Right?

Reply:
Yes. My mistake.